test/ecmascript.test.js

```diff
@@ -13,6 +13,9 @@ console.log('created directory ' + tmpdir);
 
  // These are tests known to fail so we skip them
 const ignoreCategories = new Set([
+  'Generator function.sent Meta Property',
+  'Class and Property Decorators',
+  'throw expressions',
 ]);
```

Comment:
We are skipping these tests now because they currently fail.

At first I thought this must be a regression since the tests were passing on main (head is 285c9309ee041599d31202bac3bad6bbb05aba22). However, the tests were always failing and jest was silently succeeding. Regenerating the lockfile, implicitly upgraded jest from `27.4.5` to `27.5.1` (due to the `^` prefix) and this included a fix that is now surfacing the error.

I also converted this file from `jest` to `node:test` just to confirm that it also fails on main.

Since these tests are Stage 2 features, likely no one has noticed they don't work because no JS runtime implements them.